Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -275,7 +275,14 @@ private FileSystemMetadata GetFileSystemMetadata(FileSystemInfo info)
         /// <exception cref="ArgumentNullException">The filename is null.</exception>
         public string GetValidFilename(string filename)
         {
-            var invalid = Path.GetInvalidFileNameChars();
+            // using a character list instead of GetInvalidFileNameChars, as it is not exhaustive and may not return all invalid chars
+            char[] invalid = {
```

Comment:
Move this to the class level and make it `private static readonly`

Reply:
> Move this to the class level and make it `private static readonly`

Done. Sorry about that.